pkg/Microsoft.Windows.Compatibility/test/publishcompatibilityassets.ps1

```diff
@@ -0,0 +1,60 @@
+$repoRoot = ((get-item $PSScriptRoot).parent.parent.parent.FullName);
+$winRID = "win-x64";
+$dotnetPath = -join($repoRoot, "\Tools\dotnetcli\dotnet.exe")
```

Comment:
I don't particularly like the hard-coded assumptions here but I don't know there is a better way right now.

Reply:
Maybe searching for dotnet.exe under the Tools folder? Something like how I did it to find the nupkg?